enterprise/j2ee.dd.webservice/licenseinfo.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<licenseinfo>
+    <fileset>
+        <file>src/org/netbeans/modules/j2ee/dd/impl/resources/j2ee_1_4.xsd</file>
+        <file>src/org/netbeans/modules/j2ee/dd/impl/resources/j2ee_web_services_1_1.xsd</file>
+        <file>src/org/netbeans/modules/j2ee/dd/impl/resources/j2ee_web_services_client_1_1.xsd</file>
+        <file>src/org/netbeans/modules/j2ee/dd/impl/resources/javaee_5.xsd</file>
+        <file>src/org/netbeans/modules/j2ee/dd/impl/resources/javaee_web_services_1_2.xsd</file>
+        <file>src/org/netbeans/modules/j2ee/dd/impl/resources/javaee_web_services_client_1_2.xsd</file>
+        <license ref="CDDL-1.1" />
+        <comment type="CATEGORY_B" />
+    </fileset>
+    <fileset>
+        <file>src/org/netbeans/modules/j2ee/dd/impl/resources/XMLSchema.dtd</file>
+        <file>src/org/netbeans/modules/j2ee/dd/impl/resources/datatypes.dtd</file>
+        <file>src/org/netbeans/modules/j2ee/dd/impl/resources/xml.xsd</file>
+        <license ref="W3C2" />
+        <comment>DTDs and XMLSchemata are used at runtime to offer code completion for HTML/XHTML files.</comment>
```

Comment:
The comment is correct - however it would be easier to clusters/group/filter licenses base on the comment type. The comment type `CATEGORY_B` is the superset of the description used here.

See: https://cwiki.apache.org/confluence/display/NETBEANS/Legal+FAQ